### PR TITLE
Awesome books with module

### PR DESCRIPTION
**Pull request by rbd3**
### Change made:
👉 Create index.html from the previews project
👉 Create index.css from  the previews project
👉 Create modules from the previews project
👉 Create index.js to manage all modules

### CODE REVIEW

File: modules/CurrentDate.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DateTime } from 'luxon';
+
+export default function getCurrentDate() {
+  const now = DateTime.now();
+  const formattedDate = now.toLocaleString(DateTime.DATETIME_FULL);
+  return formattedDate;
+}
```
Comment (Shisui6) [inline modules/CurrentDate.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/booksFunction.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import BookStore from './books.js';
+
+const newBookStorage = new BookStore();
+
+const bookStorage = JSON.parse(localStorage.getItem('bookCollection'));
+if (bookStorage !== null) {
+  newBookStorage.store = bookStorage;
+}
+
+export function loadBooksCollection() {
```
Comment (Shisui6) [inline modules/booksFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/booksFunction.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    button.addEventListener('click', () => {
+      const id = Number(button.getAttribute('data-book'));
+      newBookStorage.remove(id);
+      loadBooksCollection();
+      localStorage.setItem('bookCollection', JSON.stringify(newBookStorage.store));
+    });
+  });
+}
+
+export function loadContacts() {
```
Comment (Shisui6) [inline modules/booksFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/booksFunction.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  function loopingContacts(contact) {
```
Comment (Shisui6) [inline modules/booksFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: index.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import BookStore from './modules/books.js';
+import { loadBooksCollection, loadContacts } from './modules/booksFunction.js';
+import getCurrentDate from './modules/CurrentDate.js';
+
+const newBookStorage = new BookStore();
+
+const currentDateElement = document.querySelector('.datetime');
+currentDateElement.textContent += getCurrentDate();
```
Comment (Shisui6) [inline index.js]: After installing all dependencies and running your project, your webpage navigation is not working and the current date is not being displayed. Please investigate this bug and try to resolve it👍

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @rbd3, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Nice use of modules ✅
- Good design ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @rbd3 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights


✔️ Clean Codebase
✔️ Great job implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**